### PR TITLE
feat: [FTL-8612]  Added contribution guidelines

### PR DESCRIPTION
**Pull request by diegoscaro**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Some useful commands:
+
+- Use `npm run dev` to run the CLI locally.
+  > To execute commands, prefix them with `npm run dev` (and ommit the `affinidi` name).
+  > Check out the in-depth instructions in the [CLI ReadMe](README.md) file for using existing commands.
+- Run `npm test` to execute unit tests
+  > You can find them in the `src/test/unit-tests` directory.
+- Run `npm run test:integration` to execute integration tests
+  > You can find them in the `src/test/integration-test` directory.
+- `npm run lint` can be used to check for linting errors
```
Comment (aeffinidi) [inline CONTRIBUTING.md:16]: Hi
Thanks for the doc update!
I've tried to run the `test:integration`earlier
but I believe I'm missing some env vars right ?
Can we update the readme to make explain what is missing and how to get/add those infos?
Tks

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        ├── index.ts
+        ├── set.ts
+        └── get.ts
+```
+
+For simple commands, you can also take advantage of OCLIF's [command generation tool](https://oclif.io/docs/generator_commands#oclif-generate-command-name) directly in the terminal, which will scaffold a single command file in the `src/commands/` directory and a test file in `src/test/commands/` directory. Please move the created test file to the `src/test/unit-tests/commands/` directory and delete the new `src/test/commands/` directory in order to comply with this repo's test structure.
+
+## Analytics & telemetry
+
+In order to send an analytics event, use the `sendEnabledEvent()` method:
```
Comment (YoussefAWasfy) [inline CONTRIBUTING.md:40]: We only use `sendEnabledEvent()` for sign-in/up and analytics command, but for all the other commands we use `eventsControllerSend()`

File: CONTRIBUTING.md (hunk 1 above)
Comment (YoussefAWasfy) [reply CONTRIBUTING.md:16]: Yes, you need to add the API key and name space for the test email.

Comment (YoussefAWasfy) [reply CONTRIBUTING.md:16]: MAIL_API_KEY=<mail_api_key>
MAIL_NAMESPACE=<mail_namespace>

**Review by dendavidov-affinidi — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (dendavidov-affinidi) [reply CONTRIBUTING.md]: I think the external builder may run integration tests locally, but they will need to get their own MAIL_API_KEY

Comment (YoussefAWasfy) [reply CONTRIBUTING.md]: And they'll need to sign up first using this email then run the tests

**Review by YoussefAWasfy — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (diegoscaro) [reply CONTRIBUTING.md]: we will add more about tests in a future PR

**Pull request merged**